taskcluster/kinds/docker-image/kind.yml

```diff
@@ -7,5 +10,11 @@ transforms:
 
 tasks:
   browser-test: {}
-  ci: {}
+  ci:
+    args:
+      UV_VERSION: *uv_version
   rabbit-test: {}
+  python:
+    args:
+      PYTHON_VERSIONS: "3.13 3.12 3.11 3.10 3.9"
+      UV_VERSION: *uv_version
```

Comment:
This will now create a python docker image used for python client/linting tests with the python versions listed in `PYTHON_VERSIONS`.

Inspired by Taskgraph:
* https://github.com/taskcluster/taskgraph/blob/main/taskcluster/docker/python/Dockerfile
* https://github.com/taskcluster/taskgraph/blob/533a4cc6ad93ddad4ba17762066a20da86d7a47a/taskcluster/kinds/docker-image/kind.yml#L29-L33